dev/com.ibm.ws.transport.http/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -174,16 +174,22 @@ http.dispatcher=HTTP Dispatcher
 http.dispatcher.desc=HTTP Dispatcher configuration.
 
 dispatcher.trustedHeaderOrigin=Trusted private header origin
-dispatcher.trustedHeaderOrigin.desc=Private headers are used by the web server plug-in to provide information about the original \
- request. These headers take precedence over the http Host header, and are used to select a virtual host to service a request. The \
- default value is '*', which will trust incoming private headers from any source. Specify 'none' to disable private headers and rely \
- only on the http Host header, or specify a list of IP addresses to restrict private header processing to specific trusted sources.
+dispatcher.trustedHeaderOrigin.desc=Private headers are used by the web server plug-in to provide information about the original request. \
+ These headers take precedence over the HTTP Host header, and are used to select a virtual host to service a request. The default value is \
+ '*', which will trust incoming private headers from any source. Specify 'none' to disable private headers and rely only on the http Host \
+ header, or specify a comma-separated list of IP addresses and hostnames to restrict private header processing to specific trusted sources. \
+ This list supports '*' wildcards in the following formats: IP addresses cannot be shortened and must contain a value for each field, eg. \
+ "127.0.0.*" or "0:0:0:0:0:ffff:*:*", and hostnames must start with "*.", eg. "*.ibm.com". As an example, the following list is valid: \
+ "localhost, 127.0.0.1, 192.168.*.*, 0:0:0:0:0:ffff:*:*, *.ibm.com".
 
 dispatcher.trustedSensitiveHeaderOrigin=Trusted sensitive private header origin
 dispatcher.trustedSensitiveHeaderOrigin.desc=Private headers are used by the web server plug-in to provide information about the original \
```

Comment:
suggested edit:

The web server plug-in uses private headers to provide information about the original request. A subset of these headers is considered sensitive. By default, the value for this property is 'none'. Incoming sensitive private headers are not trusted from any source. To allow sensitive private header processing for specific trusted sources, specify a comma-separated list of IP addresses and hostnames. To trust incoming sensitive private headers from any source, specify '*'. This list supports '*' wildcards, with the following restrictions: IP addresses cannot be shortened and must contain a value for each field, for example "127.0.0.*" or "0:0:0:0:0:ffff:*:*", and hostnames must start with "*.", for example "*.ibm.com". The following example shows a valid list that includes wildcards: "localhost, 127.0.0.1, 192.168.*.*, 0:0:0:0:0:ffff:*:*, *.ibm.com".